galasa-ui/src/components/PageHeader.tsx

```diff
@@ -6,12 +6,16 @@
 'use client';
 
 import { Header, HeaderName , SkipToContent} from '@carbon/react';
+import styles from "../styles/Header.module.css"
 
 export default function PageHeader() {
+  
+  let galasaServiceName = process.env.NEXT_PUBLIC_GALASA_SERVICE_NAME?.trim() || "Galasa Service"
+
   return (
     <Header aria-label="Galasa Ecosystem">
       <SkipToContent />
-      <HeaderName prefix="">Galasa Ecosystem</HeaderName>
+      <HeaderName prefix="">{galasaServiceName}</HeaderName>
```

Comment:
Instead of duplicating the `galasaServiceName` variable, it would be good to pass it into the PageHeader component as a property. This will simplify the unit tests as well since we won't need to mock out the env variable.